src/js/enactors.js

```diff
@@ -308,8 +308,8 @@ fluid.defaults("uioPlus.enactor.selfVoicing", {
         controllerParentContainer: ".flc-prefs-selfVoicingWidget",
         domReaderContent: ".flc-orator-content"
     },
-    domReaderContent: ["domReaderContent", "article", "main", "genericContent"],
-    controllerParentContainer: ["controllerParentContainer", "article", "main", "genericContent"],
+    domReaderContent: ["domReaderContent", "main", "article"],
+    controllerParentContainer: ["controllerParentContainer", "main", "article"],
```

Comment:
I think we still need the `"genericContent"` option here which is likely defined in a parent grade. It provides more fallbacks to find the page content.

Reply:
You are probably looking at an old version of this file. `domReaderContent` and `controllerParentContainer` are used by subcomponents `domReader` and `controller`. As these sub-components are now disabled for turning off the page level reading. These lines have been removed in the last commit.